app/src/main/java/com/nilhcem/blenamebadge/util/Converters.kt

```diff
@@ -0,0 +1,99 @@
+package com.nilhcem.blenamebadge.util
+
+import android.content.Context
+import android.graphics.drawable.BitmapDrawable
+import android.support.annotation.DrawableRes
+import java.math.BigInteger
+import java.util.ArrayList
+
+object Converters {
+    fun convertImageToLEDHex(context: Context, @DrawableRes dId: Int): List<String> {
+        val myIcon = context.resources.getDrawable(dId)
+        val bm = (myIcon as BitmapDrawable).bitmap
+
+        val height = bm.height
+        val width = bm.width
+        val blackValue = -16777216
+
+        val image = Array(height) { IntArray(width) }
+        for (i in 0 until (height - 1)) {
```

Comment:
Using until should not need `(height - 1)`? That's why I recommended that

Reply:
Gotcha